test/navigation/AuthenticatedNavigatorSpec.scala

```diff
@@ -121,27 +120,16 @@ class AuthenticatedNavigatorSpec extends SpecBase with MockitoSugar with ScalaFu
           CheckYourAnswersController.onPageLoad()
       }
 
-      "go to Check Your Answers from ChangeWhichTaxYearsPage when using currentTaxYear" in {
-        val ua = emptyUserAnswers.set(ChangeWhichTaxYearsPage, Seq(TaxYearSelection.CurrentYear)).success.value
-          .set(ChangeWhichTaxYearsPage, Seq(CurrentYear)).success.value
-
-        navigator.nextPage(ChangeWhichTaxYearsPage, NormalMode)(ua) mustBe
-          CheckYourAnswersController.onPageLoad()
-      }
-
-      "go to Check Your Answers from ChangeWhichTaxYearsPage when not using currentTaxYear" in {
-        val ua = emptyUserAnswers.set(ChangeWhichTaxYearsPage, Seq(TaxYearSelection.CurrentYear)).success.value
-          .set(ChangeWhichTaxYearsPage, Seq(CurrentYearMinus1)).success.value
-
-        navigator.nextPage(ChangeWhichTaxYearsPage, NormalMode)(ua) mustBe
+      "go to Check Your Answers from ChangeWhichTaxYearsPage" in {
+        navigator.nextPage(ChangeWhichTaxYearsPage, NormalMode)(emptyUserAnswers) mustBe
           CheckYourAnswersController.onPageLoad()
       }
 
-      "go to HowYouWillGetYourExpenses from YourEmployer when answered true" in {
+      "go to YourAddressController from YourEmployer when answered true" in {
```

Comment:
We appear to be going to address controller from your employer, isn't Address controller pre CYA and employer post CYA?